src/index.js

```diff
@@ -0,0 +1,104 @@
+import './main.css';
+import Storage from './storage.js';
+import Store from './addremove.js';
+
+const listContainer = document.querySelector('.todo-lists');
+class Todo {
+  constructor(task, completed = false, index = 1) {
+    this.task = task;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  static sortTasks() {
+    const todos = Store.getTasks();
+    todos.sort((a, b) => {
+      const keyA = a.index;
+      const keyB = b.index;
+      if (keyA > keyB) {
+        return 1;
+      }
+      if (keyA < keyB) {
+        return -1;
+      }
+      return 0;
+    });
+    Store.editTasks(todos);
+  }
+
+  static displayTask() {
+    Todo.sortTasks();
+    const todos = Store.getTasks();
+    listContainer.innerHTML = '';
+    todos.forEach((item) => Todo.addTask(item));
+  }
+
+  static addTask(item) {
+    if (item.completed == true) {
```

Comment:
line 37 , you should use stricty equal so that it makes sure the typeof isn't automatically converted while comparing